less/base.less

```diff
@@ -49,7 +49,7 @@ body {
 
 #threads {
 	flex     : 1;
-	padding  : 1em 1.4em;
+	padding  : 0em 1.4em 1em;
```

Comment:
Without top padding the banner overlaps with the top thread buttons.

Reply:
In the previous pr, that space grew with the size of the banner. I'll try to get that behaviour back but otherwise yeah the padding needs to be put back up.

Reply:
Fixed, it grows with the banner again.